projects/lib/src/board/janusboard.cpp

```diff
@@ -92,13 +92,13 @@ Move JanusBoard::moveFromSanString(const QString& str)
 	if (hasCastlingRight(sideToMove(), KingSide))
 	{
 		Move castlingMove = WesternBoard::moveFromSanString("O-O");
-		if (str == sanMoveString(castlingMove))
+		if (castlingMove != Move() && str == sanMoveString(castlingMove))
```

Comment:
I think that instead of `castlingMove != Move()` you could write `!castlingMove.isNull()`

Reply:
Thank, you. This is much better. Changed accordingly.